zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -102,18 +107,28 @@ use crate::{types::MetaAddr, AddressBook, BoxError, PeerAddrState, Request, Resp
 pub(super) struct CandidateSet<S> {
     pub(super) peer_set: Arc<Mutex<AddressBook>>,
     pub(super) peer_service: S,
+    sleep: Sleep,
```

Comment:
Can we add a doc comment that explains the purpose of this field, or rename it to something more specific?
```suggestion
    next_peer_min_wait: Sleep,
```